internal/track/track_test.go

```diff
@@ -45,7 +45,16 @@ func makeTestRequest(uri string, request_id string, account string, orgID string
 		ctx = context.WithValue(ctx, identity.Key, identity.XRHID{
 			Identity: identity.Identity{
 				X509: identity.X509{
-					SubjectDN: "/DC=com/DC=redhat/CN=insightspipelineqe",
+					SubjectDN: "/DC=com/DC=redhat/CN=68bdb922-9e82-445c-a457-f83c13d23e3d",
```

Comment:
We should reuse the constant here?

Reply:
Agreed.  I thought about that too, but its in different packages.  :/  I guess I could export it, but that seems gross as well.  Maybe exporting it seems less gross??  I'll do that.  Hold please.